curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-functional-programming-by-building-a-spreadsheet/646d4a07a8fb14d55cd70e09.md

```diff
@@ -0,0 +1,198 @@
+---
+id: 646d4a07a8fb14d55cd70e09
+title: Step 97
+challengeType: 0
+dashedName: step-97
+---
+
+# --description--
+
+Add `someeven` and `everyeven` properties, which check if `some` or `every` element in the `nums` array is even, respectively.
```

Comment:
I think this is the first time where the `arr.some()` and `arr.every()` methods are introduced. 
If that is the case, then a brief explanation and example will be needed here

Reply:
Split into two steps and expanded the explanations.